mysql-test/t/bug96677.test

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE t1(id INT);
```

Comment:
please add percona_ prefix to testcase.

Also shouldn't we use percona bug number?

May be write upstream bug number inside the testcase

Reply:
For upstream bugs, we are using bug<upstream_id> names, as upstream